save-demo/src/main/kotlin/com/saveourtool/save/demo/controller/DiktatDemoController.kt

```diff
@@ -23,13 +25,13 @@ class DiktatDemoController(
     private val diktatDemoService: DiktatDemoService,
 ) {
     /**
-     * @param diktatRunRequest request data class with all required additional info
-     * @return [DiktatDemoResult]
+     * @param demoRunRequest request data class with all required additional info
+     * @return [DemoResult]
      */
     @PostMapping("/run")
     fun runCheckDemo(
-        @RequestBody diktatRunRequest: DiktatRunRequest,
-    ): Mono<DiktatDemoResult> = blockingToMono {
-        diktatDemoService.launch(diktatRunRequest.codeLines, diktatRunRequest.params)
+        @RequestBody demoRunRequest: DemoRunRequest,
+    ): Mono<DemoResult> = blockingToMono {
+        diktatDemoService.launch(demoRunRequest.codeLines, demoRunRequest)
```

Comment:
Is there any need to pass `codeLines` twice?

Reply:
Done